recipes/pyjc/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: pyjc
+  version: "0.1.0"
+
+source:
+  fn: pyjc-0.1.0.tar.gz
+  url: https://pypi.python.org/packages/af/9d/6a789ddbd67caf159bc548c0e99b10bcf36e7e990d3097dba0b8cb4c62f5/pyjc-0.1.0.tar.gz
```

Comment:
If the recipe is on pypi, you can use the canonical form of the `url` and then just drop the `fn` line above:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18